src/pages/spawn/SpawnVolume.tsx

```diff
@@ -38,6 +41,7 @@ export const SpawnVolume = () => {
         </BadgeWrapper>
       </TitleContainer>
       <SpawnVolumeButton showMetadata />
+      <SpawnVolumeTable volumes={volumes} />
```

Comment:
Theres something causing the volumes table to horizontally overflow.
![image](https://user-images.githubusercontent.com/4605522/94304906-5954b400-ff3e-11ea-9dcc-1ef0a5594ba7.png)
If you look at the whitespace on the right of the image